src/events/tab_event_handler.ts

```diff
@@ -64,6 +66,9 @@ export class tabEventHandler {
 		if (tabs.length === 0) return;
 		// Adding to buffer
 		this.addToBuffer(repoId, createdAt, tabs);
+	} catch(error) {
+		console.error(`[handleTabChangeEvent] Error processing tabs, error: ${error}`);
```

Comment:
@saadberry We should use CodeSyncLogger.exception so that error is logged to CW. That's the whole reason we didn't get any code-break log on the CW.